app/assets/stylesheets/components/lesson/lesson_content.scss

```
@@ -150,6 +150,14 @@
     margin-bottom: 30px;
   }
 
+  .gold-border {
```

I think naming this after what it represents would be better for readability on the curriculum files.

```suggestion
  .lesson-note {
```
I'd namespace it under `lesson-` just in case we need to use `note` for something else down the line.
 
But I can understand the reasoning behind naming it after how it looks, it fits in with how TW works.